### ISSUE TITLE
Will this create multiple PRs if multiple changes are pushed

### ISSUE THREAD

**Issue by chrux** [TreTuna/sync-branches#46]
I was looking into this project and I was wondering if a different/new PR will be created any time something is committed.

**Comment by freitasmurillo** [comment]
No. The PR is opened only once. What will happen, if the PR cannot be merged automatically, is that the opened PR will autorefresh and will contain the newest changes.

You can take a look at [this](https://github.com/lawnstarter/sync-branches/blob/7e3d71988bce7376752b89d70d8ed71cebc445f5/index.js#L51) file to better understand what I am trying to convey
